peekingduck/pipeline/nodes/draw/utils/image_processor.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+utils for image processing
+"""
+
+import cv2
+import numpy as np
+
+
+def adjust_contrast_and_brightness(frame: np.array,
+                                   contrast: float,
+                                   brightness: int) -> np.array:
+    """ Process image according to contrast or brightness setting
+
+    Args:
+        frame (np.array): image of current frame
+
+    Returns:
+        processed_frame (np.array): processed image
+    """
+    return cv2.convertScaleAbs(frame, alpha=contrast, beta=brightness)
```

Comment:
What about removing this util function and putting this cv2 method in the node directly? Doesn't seem necessary to have a separate util at this point in time.

Reply:
shifted into node itself